src/js/components/search/visualizations/geo/MapLayerToggle.jsx

```diff
@@ -41,7 +43,13 @@ const MapLayerToggle = (props) => {
                     aria-label={`Display by ${title}`}
                     data-content={title}
                     value={layer}>
-                    {title}
+                    {title === 'Congressional District' ?
+                        <div
+                            style={{ display: 'flex' }}>
+                            {title}s <FeatureFlag><TooltipWrapper tooltipPosition="right" icon="info" tooltipComponent={<CondensedCDTooltip title="Congressional District" />} /></FeatureFlag>
+                        </div>
+                        :
+                        <>{title}s </>}
```

Comment:
Now County pluralized is showing up as Countys instead of Counties.  You can create a function here to detect the string and return Countries.

Reply:
The css here needs to be tweaked because the tooltip is underlined too.
<img width="438" alt="Screenshot 2023-04-06 at 4 15 15 PM" src="https://user-images.githubusercontent.com/121010/230485029-b7d45737-d58d-462f-b8fc-2ca00037ba82.png">

Reply:
UGH.. duh. My bad. fixing

Reply:
All changes made